TaskWithClass/MyString.cpp

```diff
@@ -0,0 +1,254 @@
+#include "pch.h"
+#include "MyString.h"
+
+const char* String::c_str() const noexcept
+{
+	return m_str;
+}
+
+size_t String::length() const
+{
+	return m_len;
+}
+
+size_t String::size() const
+{
+	return m_len;
```

Comment:
Maybe it won't be optimized by compiler but we strive for excellent code and at this point let's make a size function return length(), i.e. 

return length();